src/main/java/org/broadinstitute/hellbender/tools/exome/SegmentUtils.java

```diff
@@ -414,33 +488,44 @@ public int getSite() {
-     * PRIVATE METHODS FOR READING/WRITING SEGMENT FILES                                                             *
-     *===============================================================================================================*/
-
-    private static SimpleInterval toInterval(final DataLine dataLine) {
-        return new SimpleInterval(dataLine.get(SegmentTableColumns.CONTIG.toString()),
-                dataLine.getInt(SegmentTableColumns.START.toString()), dataLine.getInt(SegmentTableColumns.END.toString()));
+    /** Simple conversion to an interval given standard segment columns
+     *
+     * @param dataLine line in a file
+     * @return interval in the dataLine.  Never {@code null}
+     */
+    public static SimpleInterval toInterval(final DataLine dataLine) {
+        Utils.nonNull(dataLine);
+        return toInterval(dataLine, SegmentTableColumns.CONTIG.toString(), SegmentTableColumns.START.toString(),
+                SegmentTableColumns.END.toString());
 
     }
 
-    private static ModeledSegment toModeledSegment(final DataLine dataLine, final boolean takeLog2) {
-        final double mean = dataLine.getDouble(SegmentTableColumns.MEAN.toString());
+    /** Simple conversion to an interval given standard segment columns
+     *
+     * @param dataLine line in a file
+     * @param contigColumn name of the contig column
+     * @param startColumn
+     * @param endColumn
+     * @return interval in the dataLine.  Never {@code null}
+     */
+    public static SimpleInterval toInterval(final DataLine dataLine, final String contigColumn, final String startColumn,
+                                            final String endColumn) {
+        Utils.nonNull(dataLine);
+        Utils.nonNull(contigColumn);
+        Utils.nonNull(startColumn);
+        Utils.nonNull(endColumn);
+        return new SimpleInterval(dataLine.get(contigColumn), dataLine.getInt(startColumn), dataLine.getInt(endColumn));
 
-        return new ModeledSegment(
-                toInterval(dataLine),
-                dataLine.get(SegmentTableColumns.CALL.toString(), ModeledSegment.NO_CALL),
-                dataLine.getLong(SegmentTableColumns.NUM_PROBES.toString()),
-                takeLog2 ? ParamUtils.log2(mean) : mean);
     }
 
-    private static <T extends Locatable> List<T> readSegmentFile(final File segmentsFile,
-                                                                 final String[] mandatoryColumns,
-                                                                 final Function<DataLine, T> dataLineToSegmentFunction) {
+    public static <T extends Locatable> List<T> readSegmentFile(final File segmentsFile,
+                                                                final String[] mandatoryColumns,
+                                                                final Function<DataLine, T> dataLineToSegmentFunction) {
```

Comment:
Can add the

`Utils.nonNull(segmentsFile);
 Utils.regularReadableUserFile(segmentsFile);`

back here.